doc/source/multiprocessing.rst

```diff
@@ -0,0 +1,38 @@
+Multiprocessing API on Ray
+==========================
+
+.. warning::
+
+  Support for the multiprocessing API on Ray is an experimental feature,
+  so it may be changed at any time without warning. If you encounter any
+  bugs/shortcomings that aren't listed in the `Known Limitations`_ section
+  below, please file an `issue on GitHub`_. Contributions are always welcome!
+
+Ray offers a subset of the `Python multiprocessing API`_ that makes it easy
+for you to scale existing applications that use multiprocessing to a cluster.
+
+Features
+--------
+
+Currently, only the ``multiprocessing.Pool`` API is supported.
+
+Usage
+-----
+
+Just replace
+``from multiprocessing import Pool`` with
+``from ray.experimental.multiprocessing import Pool`` for a drop-in replacement.
+
+If there is no Ray cluster running (i.e., started with ``ray.init``), a new one
+will be instantiated when the first ``Pool`` is created. If a local Ray cluster is
+already running, a new one won't be created.
+
+If you want to connect to an existing Ray cluster than spans multiple machines,
+just set the environment variable ``RAY_ADDRESS=<address>``. If running on the head
+node, you can use ``RAY_ADDRESS=auto``.
```

Comment:
Perhaps put an ability to specify `RAY_ADDRESS` via a parameter of some sort, falling back to the environment variable if it's unspecified? Environment variables should generally be a fallback for the user, not the preferred method of allowing a caller to customize behavior.

Reply:
Added a `ray_address` keyword argument to the `Pool` constructor.